VELOCITY_RESOLUTION: 0.1
YAWRATE_RESOLUTION: 0.1
# 预测时间2.0
PREDICT_TIME: 2.0

# 到目标终点的代价系数
TO_GOAL_COST_GAIN: 5.0
# 速度的代价系数
SPEED_COST_GAIN: 1.0
# 障碍物的代价系数
OBSTACLE_COST_GAIN: 1.0



ANGLE_RESOLUTION: 0.0087
MAX_DIST: 2.0
GOAL_THRESHOLD: 0.3
TURN_DIRECTION_THRESHOLD: 0.1
TEMP_GOAL_RADIUS: 3.0